tsl/test/expected/data_node.out

```diff
@@ -1813,17 +1898,80 @@ ORDER BY 1, 2;
  hyper_1dim      | _dist_hyper_13_25_chunk | {data_node_2,data_node_3}
 (2 rows)
 
+-- DDL should error out even if one DN is unavailable
+\set ON_ERROR_STOP 0
+ALTER TABLE hyper3 ADD COLUMN temp2 int;
+ERROR:  some data nodes are not available for DDL queries
+ALTER TABLE hyper_1dim ADD COLUMN temp2 int;
+ERROR:  some data nodes are not available for DDL queries
+\set ON_ERROR_STOP 1
+-- Mark all DNs unavailable. Metadata should still retain last DN but all
+-- activity should fail
+SELECT * FROM alter_data_node('data_node_2', available=>false);
+WARNING:  could not switch data node on 2 chunks
+  node_name  |   host    | port  |    database    | available 
+-------------+-----------+-------+----------------+-----------
+ data_node_2 | localhost | 55432 | db_data_node_2 | f
+(1 row)
+
+SELECT * FROM alter_data_node('data_node_3', available=>false);
+WARNING:  could not switch data node on 11 chunks
+  node_name  |   host    | port  |    database    | available 
+-------------+-----------+-------+----------------+-----------
+ data_node_3 | localhost | 55432 | db_data_node_3 | f
+(1 row)
+
+\set ON_ERROR_STOP 0
+INSERT INTO hyper3 VALUES ('2022-01-10 00:00:00', 1, 1);
+ERROR:  insufficient number of data nodes
+INSERT INTO hyper_1dim VALUES ('2022-01-10 00:00:00', 1, 1);
+ERROR:  insufficient number of data nodes
+UPDATE hyper3 SET temp = 10 WHERE time = '2022-01-03 00:00:00';
+ERROR:  insufficient number of data nodes
+UPDATE hyper_1dim SET temp = 10 WHERE time = '2022-01-03 00:00:00';
+ERROR:  insufficient number of data nodes
+DELETE FROM hyper3 WHERE time = '2022-01-03 00:00:00';
+ERROR:  insufficient number of data nodes
+DELETE FROM hyper_1dim WHERE time = '2022-01-03 00:00:00';
+ERROR:  insufficient number of data nodes
+SELECT count(*) FROM hyper3;
+ERROR:  Node "data_node_3" is not available for queries
```

Comment:
This error is a bit odd given that 

1. The message for INSERT/UPDATE/DELETE was different (insufficient number of data nodes)
2. All nodes are unavailable but the message singles out "data_node_3"

Reply:
We are markng all DNs as unavailable. Since data_node_3 is last it will be still associated with the chunks and hence the SELECT singles that out in the error message.

Reply:
Yeah, I understand why, just not sure it is the logical message here as opposed to something like "no data nodes available for queries"